docs/content/docs/guide/running.md

```diff
@@ -114,12 +114,12 @@ entire suite of checks once again.
 
 ![github apps rerun check](/images/github-apps-rerun-checks.png)
 
-### Gitops command on pull or merge request
+### GitOps command on pull or merge request
```

Comment:
we should probably add a rule for this in vale if that's the rules see how it's done here

Reply:
Sure